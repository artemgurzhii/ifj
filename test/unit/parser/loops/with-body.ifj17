do while bar > 0
  foo = foo * bar
  bar = bar - 1
loop
