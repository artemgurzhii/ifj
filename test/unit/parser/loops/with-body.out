(while (> (id bar) (int 0))
  (= (id foo) (* (id foo) (id bar)))
  (= (id bar) (- (id bar) (int 1))))
